yjit_iface.c

```diff
@@ -581,6 +583,8 @@ get_stat_counters(rb_execution_context_t *ec, VALUE self)
             counter_reader++;
             name_reader = name_end;
         }
+        rb_hash_aset(hash, ID2SYM(rb_intern("binding_allocation_count")), LL2NUM(binding_allocations));
+        rb_hash_aset(hash, ID2SYM(rb_intern("local_variable_set_count")), LL2NUM(local_variable_set));
```

Comment:
Instead of adding two new globals, can you add the counters to yjit_iface.h? That way they get reset by `YJIT.reset_stats!`.